CITATION.cff

```diff
@@ -0,0 +1,25 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
+cff-version: 1.2.0
+title: pixi
```

Comment:
I haven't checked the codebase to see how you handle versioning release information, but I would add the `version` key here too.

Reply:
If you decide to incorporate the version here, please make a CI check that the version is synced with the one in `Cargo.toml`, otherwise it will drift 100%.

Reply:
More an "and" than an "or" but even if you only use VCS information for version information I would use some sort of version bump tool like [`tbump`](https://github.com/scikit-hep/pyhf/blob/fc5e83018d2a2c98156e5073a6e630793a1edd55/tbump.toml) to try to go through and update information everywhere it exists.

Reply:
I would love to automate this but I want to automate the complete release and add this part to it. I've been experimenting with `release-plz` as `rattler` uses it but I haven't figure it out nicely yet. So in this PR I added the tbump as a preperation but it is not used in CI yet. 

Added the versions to the urls :+1: